app/components/Gallery.js

```diff
@@ -1,38 +1,40 @@
 
-import React, { Component } from 'react';
-import {
-  AppRegistry,
-  View,
-  ScrollView,
-  Image,
-  Text,
-  TouchableOpacity,
-  StyleSheet,
-  Dimensions
-} from 'react-native';
-
-import {
-  Card,
-  ListItem,
-  Button,
-  Icon,
-  Font,
-  Header
-} from 'react-native-elements';
-
-import {Actions} from 'react-native-router-flux';
-
-export default class Gallery extends Component {
-  render() {
-    return (
-      <ScrollView style={styles.Container}>
-
-
-        <Header
-          centerComponent={{ text: 'Welcome To Portal', style: { color: '#fff' } }}
-       />
-
 
+  import React, { Component } from 'react';
+  import {
+    AppRegistry,
+    View,
+    ScrollView,
+    Image,
+    Text,
+    TouchableOpacity,
+    StyleSheet,
+    Dimensions
+  } from 'react-native';
+
+  import {
+    Card,
+    ListItem,
+    Button,
+    Icon,
+    Font,
+    Header
+  } from 'react-native-elements';
+
+  import {
+    StackNavigator,
+  } from 'react-navigation';
+
+  import {Actions} from 'react-native-router-flux';
+
+  class HomeScreen extends React.Component {
+   
+    render() {
+      return (
+        <ScrollView style={styles.Container}>
+          <Header
+            centerComponent={{ text: 'Welcome To Portal', style: { color: '#fff' } }}
```

Comment:
You'll probably want to put these styles into a separate file at some point.

Reply:
Can you please elaborate on your comment? Do you mean we should create a separate container ? or create a separate component file for the above?

Reply:
No, just that it's generally a good idea to handle styles using separate css or scss files. It's easy to just use that ```style``` attribute while you're testing, but if you want to change your styles later, it's easier to have them all in one place, rather than hopping around to each individual component file to fix the styling on each one.

Usually there's a styles.css file, and then you give classes to each html element in your template. The element then gets its styles from whatever styles have been assigned to that class in the styles.css file.

Reply:
Great, Thanks for the feedback I will look into this.